docs/migration.md

```diff
@@ -29,6 +29,8 @@ license: |
 
 - Since 0.6.0, Celeborn changed the default value of `celeborn.client.spark.fetch.throwsFetchFailure` from `false` to `true`, which means Celeborn will enable spark stage rerun at default.
 
+- Since 0.6.0, Celeborn changed the default value of `celeborn.<module>.io.mode` from `NIO` to whether epoll mode is available, which means Celeborn will regard `EPOLL` as the default IO mode for the available epoll mode.
```

Comment:
```
- Since 0.6.0, Celeborn changed the default value of `celeborn.<module>.io.mode` from `NIO` to EPOLL if epoll mode is available, otherwise, it will fall back to NIO.
```